package.json

```diff
@@ -15,11 +17,13 @@
     "react-dom": "18.2.0"
   },
   "devDependencies": {
+    "@testing-library/react": "^13.3.0",
     "@types/node": "18.7.4",
```

Comment:
please add @testing-library/user-event - it is really useful when we would like to test a lot of component's actions

Reply:
Done.